cinema/serializers.py

```diff
@@ -22,3 +22,58 @@ def update(self, instance, validated_data):
         instance.save()
 
         return instance
+
+
+class ActorSerializer(serializers.Serializer):
+    id = serializers.IntegerField(read_only=True)
+    first_name = serializers.CharField(max_length=255)
+    last_name = serializers.CharField(max_length=255)
+
+    def create(self, validated_data):
+        return Actor.objects.create(**validated_data)
+
+    def update(self, instance, validated_data):
+        instance.first_name = validated_data.get(
+            "first_name",
+            instance.first_name
+        )
+        instance.last_name = validated_data.get(
+            "last_name",
+            instance.last_name
+        )
+        instance.save()
+        return instance
```

Comment:
I'd recommend to use ModelSerializer as these fields are related to Actor model. In this case you won't need to override create and update methods + modelserializer will automatically validate these fields